tests/ssl-opt.sh

```diff
@@ -1585,6 +1585,19 @@ CLI_OUT="cli_out.$$"
 PXY_OUT="pxy_out.$$"
 SESSION="session.$$"
 
+# If MBEDTLS_ENTROPY_C && MBEDTLS_ENTROPY_NV_SEED && !MBEDTLS_NO_PLATFORM_ENTROPY
+# create seedfile to fix test fail.
+requires_config_enabled MBEDTLS_ENTROPY_C
+requires_config_enabled MBEDTLS_ENTROPY_NV_SEED
+requires_config_disabled MBEDTLS_NO_PLATFORM_ENTROPY
+create_seedfile()
+{
+    [ -f $1 ] || dd if=/dev/urandom of=$1 bs=64 count=1 2>/dev/null
+}
+if [ "${SKIP_NEXT:-}" != "YES" ]; then
```

Comment:
Please set `SKIP_NEXT="NO"` before calling any `requires_xxx` function. And then you won't need `${:-}` here.

You still need to reset `SKIP_NEXT` to `NO` after this point since it may have been set to `YES`.